client/client.ts

```diff
@@ -164,6 +163,37 @@ function decryptResponse(t: any) {
     b.setTile(Tile.fromJSON(t));
 }
 
+/*
+ * Get signature for move proposal. This signature and the queued move will be
+ * sent to the chain for approval.
+ */
+async function proposeResponse(sig: string, uFrom: any, uTo: any) {
+    const unpackedSig: Signature = ethers.utils.splitSignature(sig);
+
+    await nStates.move(
+        formattedProof.input,
+        [
+            ...formattedProof.a,
+            ...formattedProof.b[0],
+            ...formattedProof.b[1],
+            ...formattedProof.c,
+        ],
+        unpackedSig.v,
+        unpackedSig.r,
+        unpackedSig.s
+    );
+
+    socket.emit("ping", uFrom, uTo);
+}
+
+function pingResponse(move: boolean, uFrom: any, uTo: any) {
```

Comment:
this is temporary correct? ie when you start listening on the alchemy node, the enclave won't be dependent on the client pinging to update internal belief

if it is indeed temporary, can you toss a [TMP] comment explaining

Reply:
exactly, when alchemy is set up it'll be the other way around. instead of the client "pinging" the enclave for an update, the enclave will alert the client that an update is here